app/models/in_person_enrollment.rb

```diff
@@ -126,6 +127,16 @@ def days_to_due_date
     (today...due_date).count
   end
 
+  def on_notification_sent_at_updated
+    if self.notification_sent_at && self.notification_phone_configuration
+      self.notification_phone_configuration.destroy
+    end
+  end
```

Comment:
The logic for deleting this information should also be triggered if the status is updated to cancelled.

Reply:
I'd expect cancellation logic to be added in [LG-10167,](https://cm-jira.usa.gov/browse/LG-10167) instead of this PR

Reply:
Thanks for pointing out! Good to know this is in-scope for a different story.